board/opentrons/ot2/post-build.sh

```diff
@@ -28,6 +28,7 @@ mkdir -p ${TARGET_DIR}/boot
 cp ${BINARIES_DIR}/zImage ${TARGET_DIR}/boot/zImage
 mkdir -p ${TARGET_DIR}/boot/overlays
 mkdir -p ${TARGET_DIR}/mnt/bootpart
+mv ${BINARIES_DIR}/i2c-rtc-overlay.dtb ${BINARIES_DIR}/rpi-firmware/overlays/i2c-rtc.dtbo
```

Comment:
Instead of doing this (replacing the `i2c-rtc.dtbo` file with the new one), we should probably just use the newer rpi-firmware version. That will also make sure that the overlays (and anything else being used from rpi-firmware package) are compatible with the new kernel. If we do this, we'll have to update the bt-miniuart overlay since its name has been changed in the new kernel.